### PR TITLE
update the pdf template

### PR DESCRIPTION
**Pull request by solittlework**

### CODE REVIEW

File: apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/templates.tsx [hunk 1]
```diff
@@ -256,9 +256,7 @@ export const PdfTemplates: FunctionComponent<PdfTemplatesProps> = ({ openAddTemp
 
                   setCurrentChannel(channel);
                 }}
-                bodyEditorHintText={
-                  "*GOA default styles are applied if the template doesn't include proper <html> opening and closing tags"
-                }
+                bodyEditorHintText={''}
```
Comment (jonathanweyermann) [inline apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/templates.tsx:259]: Do we not need this anymore on the main page?

Comment (jonathanweyermann) [reply apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/templates.tsx:259]: and if we don't, we should also remove the place on the main page where this text is being used, and maybe resize the page so it matches the footer/header page - see https://github.com/GovAlta/adsp-monorepo/pull/1151

**Review by jonathanweyermann — CHANGES_REQUESTED** [review]
see above

File: apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/templates.tsx (hunk 1 above)
Comment (solittlework) [reply apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/templates.tsx:259]: Ok, I see. In this case, I am going to remove it.

**Review by jonathanweyermann — APPROVED** [review]

**Pull request merged**